code/game/machinery/telecomms/presets.dm

```diff
@@ -175,6 +175,81 @@
 	freq_listening = list(COLONY_FREQ)
 	faction_shorthand = "colony"
 
+GLOBAL_LIST_EMPTY(all_static_telecomms_towers)
+
+/obj/structure/machinery/telecomms/relay/preset/tower/Initialize()
+	GLOB.all_static_telecomms_towers += src
+	. = ..()
+
+/obj/structure/machinery/telecomms/relay/preset/tower/destroy()
+	GLOB.all_static_telecomms_towers -= src
+	. = ..()
+
+/obj/structure/machinery/telecomms/relay/preset/tower/mapcomms
+	name = "TC-3T static telecommunications tower"
+	desc = "A static heavy-duty TC-3T telecommunications tower. Used to set up subspace communications lines between planetary and extra-planetary locations. Will need to have extra communication frequencies programmed into it by multitool."
+	use_power = 0
+	idle_power_usage = 10000
+	icon = 'icons/obj/structures/machinery/comm_tower3.dmi'
+	icon_state = "static1"
+	toggled = FALSE
+	bound_height = 64
+	bound_width = 64
+	freq_listening = list(COLONY_FREQ)
+
+/obj/structure/machinery/telecomms/relay/preset/tower/mapcomms/attack_hand(mob/user)
+	if(stat & NOPOWER)
+		to_chat(user, SPAN_WARNING("\The [src] makes a small plaintful beep, and nothing happens. It seems to be out of power."))
+		return FALSE
+	toggle_state(user) // just flip dat switch
+	var/turf/commloc = get_turf(src)
+	var/area/commarea = get_area(src)
+	if(on) //now, if it went on it now uses power
+		use_power = 1
+		message_admins("[key_name(user)] turned \the [src] in [commarea] ON. ([commloc ? "<a href='?_src_=admin_holder;adminplayerobservecoodjump=1;X=[commloc.x];Y=[commloc.y];Z=[commloc.z]'>JMP</a>" : "null"])")
+	else
+		use_power = 0
+		message_admins("[key_name(user)] turned \the [src] in [commarea] OFF. ([commloc ? "<a href='?_src_=admin_holder;adminplayerobservecoodjump=1;X=[commloc.x];Y=[commloc.y];Z=[commloc.z]'>JMP</a>" : "null"])")
+
+/obj/structure/machinery/telecomms/relay/preset/tower/mapcomms/attackby(obj/item/I, mob/user)
+	if(!toggled && (inoperable() || (health <= initial(health) / 2)))
+		to_chat(user, SPAN_WARNING("\The [src.name] needs repairs to have frequencies added to its software!"))
+		return
+	if(HAS_TRAIT(I, TRAIT_TOOL_MULTITOOL))
```

Comment:
```suggestion
	if(HAS_TRAIT(I, TRAIT_TOOL_MULTITOOL))
		if(!toggled && (inoperable() || (health <= initial(health) * 0.5)))
			to_chat(user, SPAN_WARNING("\The [src.name] needs repairs to have frequencies added to its software!"))
			return
```
As is, it is impossible to actually _do those repairs_ if the tower is below half health, because entire `attackby()` is stopped before it can get to `..()` for welder's interaction.
...does it even need to check for `toggled` to ask for repairs, anyway? Seems like a copypaste from the existing `toggle_state()`, and at first glance it looks like it could be an `or` instead of an `and`:
```suggestion
	if(HAS_TRAIT(I, TRAIT_TOOL_MULTITOOL))
		if(inoperable() || (health <= initial(health) * 0.5))
			to_chat(user, SPAN_WARNING("\The [src.name] needs repairs to have frequencies added to its software!"))
			return
		if(!toggled)
			to_chat(user, SPAN_WARNING("\The [src.name] needs to be toggled on to have frequencies added to its software!"))
			return
```